clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -351,8 +351,8 @@ def warn_arm_interrupt_vfp_clobber : Warning<
 def err_arm_interrupt_called : Error<
   "interrupt service routine cannot be called directly">;
 def warn_interrupt_attribute_invalid : Warning<
```

Comment:
Maybe change this name since it also contains 'signal' now?

Reply:
fixed.